@storyblok-components/video-component/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@storyblok-components/video-component",
+  "description": "Video Storyblok component",
+  "author": "Global Creative",
```

Comment:
i'm not sure about the author, it will be better if u will be the author of it right ? :)